paddlenlp/te_utils/te_helper.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
how about move this to `paddlenlp/utils/transformer_engine_utils.py`

Reply:
Done